inception-search-mtas/src/test/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndexTest.java

```diff
@@ -0,0 +1,557 @@
+import de.tudarmstadt.ukp.clarin.webanno.security.UserDao;
+import de.tudarmstadt.ukp.clarin.webanno.security.UserDaoImpl;
+import de.tudarmstadt.ukp.clarin.webanno.security.model.Role;
+import de.tudarmstadt.ukp.clarin.webanno.security.model.User;
+import de.tudarmstadt.ukp.clarin.webanno.support.ApplicationContextProvider;
+import de.tudarmstadt.ukp.dkpro.core.io.text.TextReader;
+import de.tudarmstadt.ukp.dkpro.core.io.text.TextWriter;
+import de.tudarmstadt.ukp.inception.kb.KnowledgeBaseService;
+import de.tudarmstadt.ukp.inception.kb.KnowledgeBaseServiceImpl;
+import de.tudarmstadt.ukp.inception.search.FeatureIndexingSupport;
+import de.tudarmstadt.ukp.inception.search.FeatureIndexingSupportRegistry;
+import de.tudarmstadt.ukp.inception.search.FeatureIndexingSupportRegistryImpl;
+import de.tudarmstadt.ukp.inception.search.PrimitiveUimaIndexingSupport;
+import de.tudarmstadt.ukp.inception.search.SearchResult;
+import de.tudarmstadt.ukp.inception.search.SearchService;
+import de.tudarmstadt.ukp.inception.search.SearchServiceImpl;
+import de.tudarmstadt.ukp.inception.search.index.PhysicalIndexFactory;
+import de.tudarmstadt.ukp.inception.search.index.PhysicalIndexRegistry;
+import de.tudarmstadt.ukp.inception.search.index.PhysicalIndexRegistryImpl;
+import de.tudarmstadt.ukp.inception.search.scheduling.IndexScheduler;
+import net.jodah.concurrentunit.Waiter;
+
+/**
+ * The Class MtasDocumentIndexTest.
+ */
+
+@RunWith(SpringRunner.class)
+@EnableAutoConfiguration
+@SpringBootTest(webEnvironment = WebEnvironment.MOCK)
+@EntityScan({ "de.tudarmstadt.ukp.clarin.webanno.model",
+        "de.tudarmstadt.ukp.inception.search.model",
+        "de.tudarmstadt.ukp.clarin.webanno.security.model" })
+@TestPropertySource(locations = "classpath:MtasDocumentIndexTest.properties")
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+@DataJpaTest
+@Transactional
+public class MtasDocumentIndexTest
+{
+    private final Logger log = LoggerFactory.getLogger(getClass());
+
+    // Number of miliseconds to wait for the indexing to finish. This time must be enough
+    // to allow the index be built before the query is made. Otherwise, it could affect the
+    // test results. If this happens, a largest value could allow the test to pass.
+    private final int WAIT_TIME = 3000;
+
+    private final String temporaryFolderPath = "target/MtasDocumentIndexTest";
+
+    private @Autowired UserDao userRepository;
+
+    private @Autowired ProjectService projectService;
+    private @Autowired DocumentService documentService;
+    private @Autowired SearchService searchService;
+    private @Autowired AnnotationSchemaService annotationSchemaService;
+    private @Autowired ImportExportService importExportService;
+
+    @Before
+    public void setUp()
+    {
+        userRepository.create(new User("admin", Role.ROLE_ADMIN));
+
+        FileSystemUtils.deleteRecursively(new File(temporaryFolderPath));
+    }
+
+    @Test
+    public void testSimpleQuery() throws Exception
```

Comment:
I would name this test so it describes what you actually expect, e.g. `thatSimpleDocumentCanBeIndexedAndRetreived`